fares-data-build-tool/create-data
# feat: CFD-387 Fare day end as an option for time restrictions

# Description

-   Adds fare day end as an option for time restrictions

# Testing instructions

-   Check that the fare day end option on the time restrictions page functions correctly
-   Check the validation
-   Check that you can use a fare day end time restriction in the fare journey and you generate NetEX with the correct value in the time restriction

# Type of change
